components/AddNavigationUrls.js

```diff
@@ -0,0 +1,27 @@
+const noflo = require('noflo');
+
+// Format a directions URL for Google Maps
+// https://developers.google.com/maps/documentation/urls/guide#directions-action
+function navUrl(location) {
+  const destination = `${location.latitude},${location.longitude}`;
+  const mode = 'driving';
+  const base = 'https://www.google.com/maps/dir/?api=1';
+  const url = `${base}&destination=${destination}&travelmode=${mode}`;
```

Comment:
Would be nicer to construct these URLs using the `URL` class: 
https://developer.mozilla.org/en-US/docs/Web/API/URL